frontend/src/components/user/UserApps.tsx

```diff
@@ -25,7 +29,18 @@ const UserApps: FunctionComponent = () => {
     return <Spinner size="m" text={t("loading-user-apps")} />
   }
 
-  return <ApplicationCollection title={t("your-apps")} applications={apps} />
+  const title = t(`${variant === "dev" ? "your" : "owned"}-apps`)
```

Comment:
I'm unsure if it's worth it to be smart here, being able to find translation strings via the search, might be more valuable in the long run.

Reply:
Good call, I think it will be. Updated.